logger/transport.go

```diff
@@ -145,8 +145,6 @@ func (t *transport) send(msgs []Message) error {
 		return fmt.Errorf("Server error: %d", resp.StatusCode)
 	}
 
-	// TODO(mdeltito): do we need to parse the response?
-	// we arent doing anything with it
```

Comment:
this is still an open question - if we aren't ever using it we shouldn't parse/marshal it. we don't need to decide on that here (probably more relevant for the retry work) but we need to decide one way or the other before removing this.

Reply:
agreed- think we should make a jira task of this- but not leave the comment in this public place